### PR TITLE
add routes

### PR DESCRIPTION
**Pull request by aniketsingh13**
chrome-capture-2022-3-2

### CODE REVIEW

**Review by shubhamkumar648 — APPROVED** [review]
i like the way you breakdown into component and keep all routes in one folder

File: play-app/src/Components/Main/Main.jsx [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-import headset from "../../assets/headset.jpg";
-import oculus from "../../assets/oculus.jpg";
+import Games from "../../Assets/Games.png";
+import console from "../../Assets/console.png";
+import ACCESSORIES from "../../Assets/ACCESSORIES.png";
+import VR from "../../Assets/VR.png";
+import Assasins from "../../Assets/Assasins.jpg";
+import playstation5 from "../../Assets/playstation5.png"
+import headset from "../../Assets/headset.jpg";
+import oculus from "../../Assets/oculus.jpg";
```
Comment (SanketDhabarde) [inline play-app/src/Components/Main/Main.jsx:10]: Suggestion: You can do barrel export for this

File: play-app/src/index.js [hunk 2]
```diff
@@ -1,15 +1,38 @@
...
+// makeServer();
+
+// ReactDOM.render(
+//   <React.StrictMode>
+//     <BrowserRouter>
+//     <App />
+//     </BrowserRouter>
+//   </React.StrictMode>,
+//   document.getElementById("root")
+// );
```
Comment (SanketDhabarde) [inline play-app/src/index.js]: Remove commented code

**Review by SanketDhabarde — APPROVED** [review]
- Like the idea of making separate components for routes
- You can use prettier for proper indentation
- User barrel export (like making index.js) whenever we are importing more than one file from the folder

File: play-app/src/Components/Main/Main.jsx (hunk 1 above)
Comment (aniketsingh13) [reply play-app/src/Components/Main/Main.jsx:10]: yes will change that

File: play-app/src/index.js (hunk 2 above)
Comment (aniketsingh13) [reply play-app/src/index.js]: ok will change that

**Pull request merged**